Sources/PluginViewController/Configurations/SectionsConfiguration.swift

```diff
@@ -24,10 +25,46 @@ final class SectionsConfiguration: NSObject, Configuration {
         tableViewItems = sections.map { (section: NavigationPlugin) -> PluginItem in
             return makePluginItem(for: section)
         }
+        filteredTableViewItems = tableViewItems
+    }
+
+    func filterData(with text: String) {
+        defer {
+            tableView?.reloadData()
+        }
+        guard text.isEmpty == false else {
+            filteredTableViewItems = tableViewItems
+            return
+        }
+
+        var resultPlugins: [Plugin] = []
+        tableViewItems.forEach { pluginItem in
+            resultPlugins.append(contentsOf: searhPlugins(in: pluginItem.plugin, withText: text))
+        }
+        let resultPluginItems = resultPlugins.map { plugin -> PluginItem in
+            PluginItem(title: plugin.title.string, plugin: plugin, children: [])
+        }
+        filteredTableViewItems = [PluginItem(title: "Search result",
+                                             plugin: SectionPlugin(plugins: resultPlugins),
+                                             children: resultPluginItems)]
```

Comment:
This logic will run on every text change. You can optimize it by flatting plugins in `configure` deboogger method